dom-events/buttons.js

```diff
@@ -0,0 +1,31 @@
+// Click Event
+function handleClick(event) {
+  console.log('button clicked');
+  console.log('Event:', event);
+  console.log('Event Target:', event.target);
+}
+
+const singleClickButton = document.querySelector('.click-button');
+
+singleClickButton.addEventListener('click', handleClick);
```

Comment:
Don't forget the `$` in front of your variable names! Whenever we store a DOM element in a variable, we want to include that dollar sign!